scripts/fetch/getNMdata.R

```diff
@@ -0,0 +1,12 @@
+#' Get NM daily data
+#'
+#'@param viz the vizlab object 
+fetcher.getNMdata <- function(viz = getContentInfo(viz.id = "getNMdata")){
```

Comment:
Ah is that just a way to nicely catch issues with the viz objects and throw helpful errors?